src/EventListener/HeaderReplayStopPropagationListener.php

```diff
@@ -18,40 +18,30 @@
 /**
  * Class HeaderReplayStopPropagationListener.
  *
- * This listener is registered with a very high priority on both, the
- * kernel.response and the kernel.terminate listeners. It makes sure
- * it stops propagation of these events if the current request is a preflight
- * request. It never makes sense to modify a preflight request by one of these
- * events as the bundle provides its own event for this.
+ * This listener is registered with a very high priority on the kernel.terminate
+ * listeners. It makes sure it stops propagation of these events if the current
```

Comment:
`events. It makes sure to stop propagation of the event if the current`

Reply:
Updated the docs.